### PR TITLE
all

### PR DESCRIPTION
**Pull request by avivpxi**

### CODE REVIEW

File: api.go [hunk 1]
```diff
@@ -0,0 +1,371 @@
+// Copyright 2024 HUMAN. All rights reserved.
```
Comment (massdosage) [inline api.go:1]:
```suggestion
// Copyright 2024 HUMAN Security.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [[0.0.1](https://github.com/PerimeterX/envite/releases/tag/v0.0.1)] - 2023-12-21
```
[the remark this answers is not in the document]
Comment (avivpxi) [reply CHANGELOG.md:8]: i do want to publish everything this week so nuking everything would be too much. this repo was created with open source in mind and did not contain any private stuff. i'll look into adding the headers in previous versions, or consider removing the older tags and history.
i remembered we agreed that new open repos should be created as private first, but i can't find it written anywhere...not sure what we settled on

File: static_files.go [hunk 3]
```diff
@@ -0,0 +1,524 @@
+// Code generated by go-bindata. (@generated) DO NOT EDIT.
```
[the remark this answers is not in the document]
Comment (avivpxi) [reply static_files.go:1]: yes, this is fine.

in more detail: the go compiler does not support hooks (like pre-compile) for security reasons. this requires manually generating code before compiling. if a consumer of such library had to do it themselves, it means pulling dependencies, then find the path to a centralized registry on their machine, find the library location, generate code and only then compile. they went for the practice of committing generated code into the source code instead (another source for your reference - https://stackoverflow.com/a/56417892/1932186)

File: CHANGELOG.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (avivpxi) [reply CHANGELOG.md]: removed older git tags and releases (there are leftovers in the main branch commit history, although we have no private stuff there)
fixed all other issues.
let me know

[the remark this answers is not in the document]
Comment (massdosage) [reply CHANGELOG.md]: > i do want to publish everything this week so nuking everything would be too much. this repo was created with open source in mind and did not contain any private stuff. i'll look into adding the headers in previous versions, or consider removing the older tags and history. i remembered we agreed that new open repos should be created as private first, but i can't find it written anywhere...not sure what we settled on

Yeah, usually they should be private first and then only made public after review. The alternative is to create something as open right from the start which is also fine, we just need to announce that internally and agree on it beforehand.

File: LICENSE [hunk 4]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2024 HUMAN Security
```
Comment (massdosage) [inline LICENSE:3]:
```suggestion
Copyright (c) 2024 HUMAN Security.
```

Comment (avivpxi) [reply LICENSE]: ugh sorry, fixed 🙏 
i'm closing this one, we can open another as needed

**Pull request closed without merging**